packages/compiler/src/types.ts

```diff
@@ -13,6 +14,13 @@ export interface CompilerInitAction {
   preactVersion: string;
 }
 
+export interface CompilerSetLocalComponentAction {
+  action: 'set-local-components';
+  components: { componentPath: string; componentSource: string }[];
+  resetCache?: boolean;
```

Comment:
🔪

Reply:
Good catch! `resetCache` was leftover from my first draft and it slipped through the cracks.